engine/src/test/java/com/google/android/fhir/sync/SyncJobTest.kt

```diff
@@ -198,4 +199,53 @@ class SyncJobTest {
 
     job.cancel()
   }
+
+  @Test
+  fun `sync time should update on every sync call`() = runBlockingTest {
+    val worker1 = PeriodicWorkRequestBuilder<TestSyncWorker>(15, TimeUnit.MINUTES).build()
+
+    // get flows return by work manager wrapper
```

Comment:
```suggestion
    // Get flows return by work manager wrapper
```

and for other comments too.

Reply:
Done